src/Microsoft.Dnx.Testing.Abstractions/ITestHostServices.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Dnx.Testing.Abstractions
+{
+    public interface ITestHostServices
```

Comment:
:+1:

Should we just build a static accessor for this thing now a la `PlatformServices`. Really strange for unit testing to have DI when console apps don't.